components/recurring-contributions/RecurringContributionsCard.js

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { get } from 'lodash';
+import { withRouter } from 'next/router';
+import { defineMessages, useIntl } from 'react-intl';
+
+import Avatar from '../Avatar';
+import Container from '../Container';
+import FormattedMoneyAmount from '../FormattedMoneyAmount';
+import { Flex } from '../Grid';
+import I18nCollectiveTags from '../I18nCollectiveTags';
+import LinkCollective from '../LinkCollective';
+import StyledButton from '../StyledButton';
+import StyledCard from '../StyledCard';
+import StyledTag from '../StyledTag';
+import { P } from '../Text';
+import { withUser } from '../UserProvider';
+
+import RecurringContributionsPopUp from './RecurringContributionsPopUp';
+
+const getBackground = collective => {
+  const backgroundImage = collective.backgroundImageUrl || get(collective, 'parentCollective.backgroundImageUrl');
+  const primaryColor = get(collective.settings, 'collectivePage.primaryColor', '#1776E1');
+  return backgroundImage
+    ? `url(/static/images/collective-card-mask.svg) 0 0 / cover no-repeat, url(${backgroundImage}) 0 0 / cover no-repeat, ${primaryColor}`
+    : `url(/static/images/collective-card-mask.svg) 0 0 / cover no-repeat, ${primaryColor}`;
+};
+
+const messages = defineMessages({
+  amountContributed: {
+    id: 'Subscriptions.AmountContributed',
```

Comment:
Still `Subscriptions` here, let's change it now because if we do it later, people would have to re-translate the string.